backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -46,11 +46,17 @@ import { checkOptInCode, activationLink, printTimeDuration } from './UserResolve
 import { sendAccountActivationEmail } from '@/mailer/sendAccountActivationEmail'
 import { transactionLinkCode as contributionLinkCode } from './TransactionLinkResolver'
 import CONFIG from '@/config'
+import { getUserCreation } from './util/getUserCreation'
+import { isContributionValid } from './util/isContributionValid'
 
 // const EMAIL_OPT_IN_REGISTER = 1
 // const EMAIL_OPT_UNKNOWN = 3 // elopage?
 const MAX_CREATION_AMOUNT = new Decimal(1000)
-const FULL_CREATION_AVAILABLE = [MAX_CREATION_AMOUNT, MAX_CREATION_AMOUNT, MAX_CREATION_AMOUNT]
+export const FULL_CREATION_AVAILABLE = [
```

Comment:
Can we have this in a `constants` folder?

Reply:
Sure do we have one or should I set the const folder on same level as the util one?